spec/wizards/a_level_steps/are_any_a_levels_required_for_this_course_spec.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe ALevelSteps::AreAnyALevelsRequiredForThisCourse do
+  subject(:wizard_step) { described_class.new }
+
+  describe 'validations' do
+    it 'is valid with a valid answer' do
+      wizard_step.answer = 'yes'
+      expect(wizard_step).to be_valid
```

Comment:
Shall we check all the valid answers?
```suggestion
      wizard_step.answer = 'yes'
      expect(wizard_step).to be_valid
      
      wizard_step.answer = 'yes'
      expect(wizard_step).to be_valid
```

or something like `wizard_step.answer = ['yes', 'no'].sample`

Otherwise this test gives the impression that yes is the only valid answer.

Reply:
Yes, done